docs/pages/api-reference/operators.md

```diff
@@ -236,4 +236,24 @@ The `assign` operator has the following parameters:
 Fennel does not allow you to assign keys or timestamp columns from a dataset.
 :::
 
+### Groupby/ Window
 
+In Fennel, the `window` operator enables grouping events into windowed streams based on specified criteria. A common application of this operator is in creating sessions, where it forms a window comprising events that occur in a session.
+The `window` operator must be preceded by `groupby` to identify the fields that will be used for grouping the events before they are windowed.
+
+`groupby` takes the following parameters:
+
+1. `*args: str` - positional arguments that specify the list of fields to group by.
+
+The `window` operator takes the following parameters:
+
+1. ``type: str``: positional argument, that specifies the type of windows to create. Allowed types
+   are `session`, `tumble` and `sliding`.
+2. ``gap: str``: positional argument, that specifies the maximum time gap between two consecutive events for clubbing them into a single window.
+3. ``field: str``: positional argument, that specifies the name of the keyed window field. The window field will contain following information: begin time, end time and number of events in a particular window.
```

Comment:
Should be a kwarg, not positional arg. Also please specify that a field with name will be created in the output dataset with type `Window` with a link to some info about the `Window` struct (or put that in-line itself).